datafusion/functions/src/regex/regexpmatch.rs

```diff
@@ -119,7 +119,7 @@ fn get_regexp_match_doc() -> &'static Documentation {
     DOCUMENTATION.get_or_init(|| {
         Documentation::builder()
             .with_doc_section(DOC_SECTION_REGEX)
-            .with_description("Returns a list of [regular expression](https://docs.rs/regex/latest/regex/#syntax) matches in a string.")
+            .with_description("Returns the first [regular expression](https://docs.rs/regex/latest/regex/#syntax) matche in a string.")
```

Comment:
matche ?

Reply:
Oh, i'll fix that rlly quick.

Reply:
fixed in https://github.com/apache/datafusion/pull/13014